service_common.go

```diff
@@ -291,9 +303,7 @@ func (c *controller) addServiceBinding(svcName, svcID, nID, eID, containerName s
 
 	// Add loadbalancer service and backend in all sandboxes in
```

Comment:
maybe this comment now is outdated. It should be in the load balancer sandbox only, not all sandboxes